lib/src/base.dart

```diff
@@ -980,6 +985,7 @@ class Browser {
 
   /// Produces a [Map] that can be serialized to JSON.
   Map<String, dynamic> toJson() {
+    // ignore: omit_local_variable_types
     final Map<String, dynamic> json = {};
```

Comment:
If the type is left to be inferred here, tests fails:

```
TypeError: Instance of 'JsLinkedHashMap<dynamic, dynamic>': type 'JsLinkedHashMap<dynamic, dynamic>' is not a subtype of type 'Map<String, dynamic>'
```

Reply:
This didn't work?

```dart
final json = <String, dynamic>{};
```

Reply:
I can see how `final json = {};` doesn't work because there's no type anywhere at all, which causes it to default to `Map<dynamic, dynamic>`.